src/pages/scripts/tutorials/tips-tricks/index.md

```diff
@@ -0,0 +1,56 @@
+# Tips and tricks for UXP scripts
+Additional tips and tricks that will help you while developing UXP Scripts.
+
+## Scripts panel
+
+The InDesign Scripts panel is the easiest and best way to run most InDesign scripts. If the panel is not already visible, you can display it by choosing Window > Utilities > Scripts.
+
+### Editing scripts
+To edit a script shown in the Scripts panel, select the script and choose Edit Script from the Scripts panel menu. Alternatively, you can hold down the Option (Mac OS) or Alt (Windows) key and double-click the script's name. This opens the script in the editor you defined for the script file type.
+
+### Open folder
+To open the folder containing a script shown in the Scripts panel, select the script and choose Reveal in Finder (Mac OS) or Reveal in Explorer (Windows). Alternatively, you can hold down the Command (Mac OS) or Ctrl-Shift (Windows) keys and double-click the script's name. The script folder opens in the Finder (Mac OS) or Explorer (Windows).
+
+### Keyboard shortcuts
+
+To add a keyboard shortcut for a script, choose Edit > Keyboard Shortcuts, select an editable shortcut set from the Set menu, then choose Product Area > Scripts. A list of the scripts in your Scripts panel appears. Select a script and assign a keyboard shortcut as you would for any other InDesign feature.
```

Comment:
@anoopvalomkot  The 'Set' value should be '[Default]'. Please confirm.

Reply:
yes